packages/components/src/styles/media/media.stories.ts

```diff
@@ -35,25 +36,21 @@ export const Default = {
   }
 };
 
-export const MediaSample = {
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: `
-          <figure class="%CLASSES% max-w-xl p-4">
-            <div class="sd-copyright" style="--copyright: '© 2024 Solid Design System';">
-              <img src="./placeholders/images/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover"/>
-            </div>
-            <figcaption>%SLOT%</figcaption>
-          </figure>
-      `
-      },
-      args
-    });
-  }
+/**
+ * Use the `sd-media--inverted` class when displayed on primary background.
+ */
+
+export const Inverted = {
+  render: () => html`
+    <div class="p-4 bg-primary max-w-xl">
+      <figure class="sd-media sd-media--inverted max-w-xl p-4">
```

Comment:
Also not sure about the width of the image. Is it required for some reason?

Reply:
Not really. Updated